examples/TurnOnFujitsuAC/TurnOnFujitsuAC.ino

```diff
@@ -0,0 +1,44 @@
+// Copyright 2017 Jonny Graham
+#include <IRsend.h>
+#include <ir_Fujitsu.h>
+
+IRFujitsuAC fujitsu(1);  // IR led controlled by Pin D1.
```

Comment:
FYI, I'm fairly sure Pin D1 == GPIO 5. Pin 1 == GPIO 1 == TXD0, which is a bad choice.
i.e. Change (1) to (5).